ws-nextjs-app/pages/[service]/send/[id]/types.ts

```diff
@@ -0,0 +1,54 @@
+import { Services, Variants } from '#app/models/types/global';
+import { ParsedUrlQuery } from 'querystring';
+
+export type InputProps = {
+  id: string;
+  name: string;
+};
+
+export interface PageDataParams extends ParsedUrlQuery {
+  id: string;
+  page?: string;
+  service: Services;
+  variant?: Variants;
+  // eslint-disable-next-line camelcase
+  renderer_env?: string;
+  resolvedUrl: string;
+}
```

Comment:
I know this is just a spike, but are any of these `PageDataParams` common between any of the NextJS pages, and could be reused? (looks like a dupe of https://github.com/bbc/simorgh/blob/17d50a729d5736c70b4c1439e701796d2a95a437/ws-nextjs-app/pages/%5Bservice%5D/live/%5Bid%5D/%5B%5B...variant%5D%5D.page.tsx#L31-L39